apps/backend/src/sales/dto/sales-wip.dto.ts

```diff
@@ -0,0 +1,77 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsNotEmpty,
+  IsString,
+  ArrayMinSize,
+  IsArray,
+  ValidateNested,
+  IsNumber,
+  IsDefined,
+  ArrayUnique
+} from 'class-validator';
+import { PaymentsDTO } from './payment.dto';
+import { Type } from 'class-transformer';
+
+export class SalesReconciliationDTO {
+  @ApiProperty({
+    description: 'id',
+    example: '20230103-00025-1000001'
+  })
+  @IsString()
+  @IsNotEmpty()
+  id!: string;
+
+  @ApiProperty({
+    description: 'Agent No.',
+    example: '0025'
+  })
+  @IsString()
+  @IsNotEmpty()
+  location!: string;
+
+  @ApiProperty({
+    description: 'Sales Txn Date',
+    example: '2022-11-30-13.33.31.875973'
+  })
+  @IsString()
```

Comment:
Date or timestamp? Can we add validation on the expected format since we control this api?

Reply:
**TODO for the sales api:

set timezone params 
- either by docs
- or by js-timezones